src/test/java/com/dabsquared/gitlabjenkins/publisher/GitLabCommitStatusPublisherTest.java

```diff
@@ -219,6 +236,62 @@ private HttpRequest prepareExistsCommit(String projectId, String sha) throws Uns
                 .withHeader("PRIVATE-TOKEN", "secret");
     }
 
+    private HttpRequest prepareGetProjectResponse(String projectName, int projectId) throws UnsupportedEncodingException {
+      HttpRequest request= request()
+               .withPath("/gitlab/api/v3/projects/" + URLEncoder.encode(projectName, "UTF-8"))
+               .withMethod("GET")
+               .withHeader("PRIVATE-TOKEN", "secret");
+
+        HttpResponse response = response().withBody(
+            "{\"id\":"+projectId+","+
```

Comment:
Please add a space before and after each plus sign. Or even better move the JSON to a separate file with placeholders like in the `MergeRequestBuildActionTest`.